cmd/tracee-rules/output.go

```diff
@@ -104,3 +117,47 @@ func sendToWebhook(t *template.Template, res detect.Finding, webhook string, web
 	_ = resp.Body.Close()
 	return nil
 }
+
+func sendToFluentForward(t *template.Template, res detect.Finding, fluentForwardUrl string, fluentForwardTemplate string, fluentForwardTag string) error {
+	if fluentForwardTag == "" {
+		return errors.New("error sending to Forward: --forward-tag flag is required when using --forward-url flag")
+	}
+
+	// Convert to payload in appropriate format
+	var payload string
+
+	switch {
+	case fluentForwardTemplate != "":
+		if t == nil {
+			return fmt.Errorf("error writing to template: template not initialized")
+		}
+		buf := bytes.Buffer{}
+		if err := t.Execute(&buf, res); err != nil {
+			return fmt.Errorf("error writing to the template: %v", err)
+		}
+		payload = buf.String()
+	default:
+		return errors.New("error sending to Forward: --forward-template flag is required when using --forward-url flag")
+	}
+
+	c := client.New(client.ConnectionOptions{
+		Factory: &client.ConnFactory{
+			Address: fluentForwardUrl,
+		},
+	})
+
+	if err := c.Connect(); err != nil {
+		return fmt.Errorf("error connecting to Forward URL %q: %v", fluentForwardUrl, err)
+	}
+	defer c.Disconnect()
+
+	record := map[string]interface{}{
+		"event": payload,
```

Comment:
buf.String() ?

Reply:
possibly but then it keeps `buf` in scope so will leave for review